src/librustdoc/json/import_finder.rs

```diff
@@ -0,0 +1,30 @@
+use rustc_data_structures::fx::FxHashSet;
+use rustc_hir::def_id::DefId;
+
+use crate::{
+    clean::{self, Import, ImportSource, Item},
+    fold::DocFolder,
+};
+
+pub(crate) fn get_imports(krate: clean::Crate) -> (clean::Crate, FxHashSet<DefId>) {
```

Comment:
Please add a small comment explaining what this function does.

Reply:
Done